components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -60,6 +88,20 @@ class NewTabPage extends React.Component<Props, State> {
     }
   }
 
+  refreshActions = () => {
+    this.props.actions.getGrants()
```

Comment:
Why do we need to call a reducer here that doesn't change state? It will cause unnecessary redux and react events and re-rendering. Why not simply call `chrome.braveRewards.getGrants()` here?

Reply:
done

Reply:
done